src/constants/tooltips/trade.ts

```diff
@@ -141,7 +143,13 @@ export const tradeTooltips = {
   }),
   'max-reward-sept-2025': ({ stringGetter }) => ({
     title: stringGetter({ key: TOOLTIP_STRING_KEYS.MAXIMUM_REWARDS_SEPT_2025_TITLE }),
-    body: stringGetter({ key: TOOLTIP_STRING_KEYS.MAXIMUM_REWARDS_SEPT_2025_BODY }),
+    body: stringGetter({
+      key: TOOLTIP_STRING_KEYS.MAXIMUM_REWARDS_SEPT_2025_BODY,
```

Comment:
I don't love that this tooltip is labeled as `sept`. Should we just make it a generic reward string